### ISSUE TITLE
Custom email per instance

### ISSUE THREAD

**Issue by jb-fol** [mdjnelson/moodle-mod_customcert#569]
Hey Mark - I often use the Custom Certificate module for vouchers, membership cards, etc, as well as certificates. The generic email sent to students is editable in the language strings but it's really difficult to get wording that works for all scenarios.

It would be mega-helpful to have a setting for each certificate instance that would allow the email message to be customised. Similar to the reengagement module for example, which has Notification subject (User) and Notification content (User). 

Would that be doable?

**Comment by mdjnelson** [comment]
Yep, this is doable but would take quite a lot of development time on my behalf.

**Comment by mdjnelson** [comment]
Closing due to inactivity as part of issue housekeeping. I’m a solo maintainer with limited time right now, so I’m focusing on issues with recent activity and clear next steps.

If this is still relevant, please comment with an update (bug: current repro steps + environment; request: use case + desired outcome). With updated info, I can re-triage and reopen or track it in a new issue if needed.
